cypress/integration/linkingAccessionIds.spec.ts

```diff
@@ -132,49 +113,43 @@ describe("Linking Accession Ids", function () {
       .as("runAccessionId")
 
     // Analysis form
-    cy.get("div[role=button]").contains("Analysis").click()
-    cy.get("div[aria-expanded='true']")
-      .siblings()
-      .within(() =>
-        cy
-          .get("div[role=button]")
-          .contains("Fill Form", { timeout: 10000 })
-          .should("be.visible")
-          .then($btn => $btn.click())
-      )
+    cy.clickFillForm("Analysis")
     cy.get("[data-testid='title']").type("Test Analysis title")
     cy.get("select[data-testid='analysisType']").select("Reference Alignment")
     cy.get("select[data-testid='analysisType.referenceAlignment.assembly']").select("Standard")
-    cy.get("input[data-testid='analysisType.referenceAlignment.assembly.accessionId']").type("Standard accessionId")
+    cy.get("input[data-testid='analysisType.referenceAlignment.assembly.accession']").type("Standard accessionId")
+
     // Select studyAccessionId
-    cy.get("select[data-testid='studyRef.accessionId']").then(($el: any) => {
-      const studyAccessionId = cy.get("@studyAccessionId")
-      $el.select(studyAccessionId)
-    })
+    cy.get("@studyAccessionId").then(id =>
+      cy.get("select[data-testid='studyRef.accessionId']").select(`${id} - File name: testFile.xml`)
+    )
+    // cy.get("select[data-testid='studyRef.accessionId']").then(($el: any) => {
```

Comment:
we can remove this if not needed, or is there an issue and we should keep it for reference ?

Reply:
My bad. Yes it should be removed.